src/QMCWaveFunctions/LCAO/SoaAtomicBasisSet.h

```diff
@@ -656,6 +663,47 @@ struct SoaAtomicBasisSet
       }
     }
   }
+  void createResource(ResourceCollection& collection) const
+  {
+    auto resource_index = collection.addResource(std::make_unique<SoaAtomicBSetMultiWalkerMem>());
```

Comment:
Since you don't consume the return. Just don't collect the return.

Reply:
fixed